packages/flutter_tools/lib/src/windows/migrations/version_migration.dart

```diff
@@ -0,0 +1,97 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../../base/file_system.dart';
+import '../../base/project_migrator.dart';
+import '../../cmake_project.dart';
+
+const String _cmakeFileBefore =
+'# Apply the standard set of build settings. This can be removed for applications\r\n'
+'# that need different build settings.\r\n'
+'apply_standard_settings(\${BINARY_NAME})\r\n'
```

Comment:
Wouldn't we expect `${BINARY_NAME}` to have been interpolated?

Reply:
`BINARY_NAME` is a CMake variable, not a Dart variable. Here's an example CMakeLists.txt file that has already been migrated:

https://github.com/flutter/flutter/blob/2379b18f8db2fae3bc8f189c7c24e98f6fdff9aa/examples/hello_world/windows/runner/CMakeLists.txt#L21

Reply:
Ahh, lol, that makes sense.